### DISCUSSION TITLE
Is it possible to support MinGW?

### DISCUSSION THREAD

**Discussion by ognevny** [microsoft/sudo#92] [General]
### Description of the new feature / enhancement

support for MinGW (*-pc-windows-gnu{,llvm} targets)

### Scenario when this would be used?

this can be used (if possible) for work under msys2 (for example) environments

**Comment by joadoumie** [comment]
Are you trying to build the project in MinGW? This is good, we've never tested this. You are welcome to try it. If it requires large changes to the project, we will not be able to support this.
